
From: Hugh Dickins <hugh@veritas.com>

3c59x now uses generic_mii_ioctl, so VORTEX should select MII.

Signed-off-by: Hugh Dickins <hugh@veritas.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/Kconfig |    1 +
 1 files changed, 1 insertion(+)

diff -puN drivers/net/Kconfig~3c59x-vortex-select-mii drivers/net/Kconfig
--- 25/drivers/net/Kconfig~3c59x-vortex-select-mii	Mon Oct  4 14:48:32 2004
+++ 25-akpm/drivers/net/Kconfig	Mon Oct  4 14:48:32 2004
@@ -691,6 +691,7 @@ config ELMC_II
 config VORTEX
 	tristate "3c590/3c900 series (592/595/597) \"Vortex/Boomerang\" support"
 	depends on NET_VENDOR_3COM && (PCI || EISA)
+	select MII
 	---help---
 	  This option enables driver support for a large number of 10mbps and
 	  10/100mbps EISA, PCI and PCMCIA 3Com network cards:
_
